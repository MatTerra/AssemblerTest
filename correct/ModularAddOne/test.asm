BEGIN
section data
test: SPACE
section text
PUBLIC test
END
