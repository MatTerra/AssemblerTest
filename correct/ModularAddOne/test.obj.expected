H: test
H: 1
H: 0
H: D test 0
T:  0
